js/manager.js

```diff
@@ -149,6 +149,29 @@ const DFPManager = Object.assign(new EventEmitter().setMaxListeners(0), {
     }
   },
 
+  destroySlots(slotsToDestroy) {
+    if (managerAlreadyInitialized === true) {
+      loadAlreadyCalled = false;
+      this.getGoogletag().then((googletag) => {
+        if (slotsToDestroy !== undefined) {
+          googletag.destroySlots(
+            Object.keys(slotsToDestroy).map(slotId => slotsToDestroy[slotId].gptSlot),
+          );
+        } else {
+          googletag.destroySlots();
+        }
+      });
+    }
+  },
+
+  setNewAdUnit(slots, newAdUnit) {
```

Comment:
- can u please update the documentation?.
- it would be great to add a small testcase for this.